internal/pkg/cli/local_run_test.go

```diff
@@ -0,0 +1,260 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/aws/copilot-cli/internal/pkg/cli/mocks"
+	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+)
+
+var testError = errors.New("some error")
+
+type localRunAskMocks struct {
+	store       *mocks.Mockstore
+	ws          *mocks.MockwsWlDirReader
+	prompt      *mocks.Mockprompter
+	deployStore *mocks.MockdeployedEnvironmentLister
+}
+
+func TestLocalRunOpts_Validate(t *testing.T) {
+	testCases := map[string]struct {
+		wantedError error
+	}{
+		"no app in workspace": {
```

Comment:
Can we add two more test cases:
1. "fail to read the application from SSM store" with a `m.store.Expect().GetApplication().Return(nil, testError)`
2. "success"